CHANGELOG.md

```diff
@@ -3,7 +3,64 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 1.1.0
+
+This release contains a mix of features, performance improvements, and bugfixes.
+
+### Topdown and Rego
+
+- rego: Fixing broken `BenchmarkCustomFunctionInHotPath` ([#7312](https://github.com/open-policy-agent/opa/pull/7312)) authored by @johanfylling
```

Comment:
A bit silly perhaps, but we _could_ start off on a more positive note 😄

Reply:
But that would mess up the alphabetic order 😢.
We can drop it, it's not front facing 😄.